src/analysis/sampling/hit_and_run.jl

```diff
@@ -47,33 +47,70 @@ samples = hit_and_run(100_000, opt_model; keepevery=10, samplesize=5000)
 ```
 """
 function hit_and_run(
-    N::Int,
-    opt_model;
+    model,
+    optimizer;
+    modifications = [],
+    N = 1000,
     keepevery = _constants.sampling_keep_iters,
     samplesize = _constants.sampling_size,
-    random_objective = false,
+    warmup_indices = collect(1:n_reactions(model)),
+    workers = [myid()],
+    nchains = 1,
 )
 
-    lbs, ubs = get_bound_vectors(opt_model) # get actual ub and lb constraints, can't use model function because the user may have changed them in the function arguments
+    # get warmup points in parallel, similar to FVA
+    ws, lbs, ubs = warmup(
+        model,
+        optimizer;
+        modifications = modifications,
+        warmup_points = warmup_indices,
+        workers = workers, # parallel
+    )
+
+    # load warmup points to workers
+    map(fetch, save_at.(workers, :ws, Ref(:($ws))))
```

Comment:
this can be done more easily in one step, you'd do something like:

```
save_at.(workers, :_cobrexa_hit_and_run_warmup, Ref(:($ws, $lbs, $ubs)))
```

btw please use the _cobrexa_prefixed_symbols to avoid name collisions (it's saved into the global namespace on remotes).